bundles/org.eclipse.equinox.servletbridge/src/org/eclipse/equinox/servletbridge/FrameworkLauncher.java

```diff
@@ -896,9 +900,13 @@ protected int findMax(String[] candidates) {
 		for (int i = 0; i < candidates.length; i++) {
 			String name = candidates[i];
 			String version = ""; //$NON-NLS-1$ // Note: directory with version suffix is always > than directory without version suffix
-			int index = name.indexOf('_');
-			if (index != -1)
-				version = name.substring(index + 1);
```

Comment:
This would then become

			int index = name.indexOf(VERSION_SEP_UNDERSCORE);
			if (index != -1) {
				index = name.indexOf(VERSION_SEP_DASH);
			}
			if (index != -1) {
				version = name.substring(index + 1);
			}